db/migrate/20151207233640_remove_duration_add_arival_to_flight.rb

```diff
@@ -2,7 +2,7 @@ class RemoveDurationAddArivalToFlight < ActiveRecord::Migration
   def up
     remove_column :flights, :duration
     rename_column :flights, :departure, :departure_datetime
-    add_column :flights, :flight_no, :string, after: :airline
+    # add_column :flights, :flight_no, :string, after: :airline
```

Comment:
Since migrations use timestamps, it is more efficient to create another migration file instead of commenting out a line in an existing file